src/vs/workbench/parts/search/browser/openAnythingHandler.ts

```diff
@@ -244,7 +274,7 @@ export class OpenAnythingHandler extends QuickOpenHandler {
 		return null;
 	}
 
-	public getResultsFromCache(searchValue: string, range: IRange = null): QuickOpenEntry[] {
+	public getResultsFromCache(searchValue: string, range: IRange = null) {
```

Comment:
One of our coding principles is to always type the return value of a function, even if TS can infer it. So please add an interface for this method too.